File: HTML/src/main/java/it/polimi/tiw/wt_music_playlist_2025/entity/Track.java
package it.polimi.tiw.wt_music_playlist_2025.entity;

public record Track(
        int id,
        String filePath,
        String imagePath,
        String title,
        String author,
        String albumTitle,
        int albumPublicationYear,
        String genre,
        int position
) implements Entity {

}

File: JS/src/main/webapp/model/track.ts
export class Track {
  id!: number;
  loaderId!: number;
  filePath!: string;
  imagePath!: string;
  title!: string;
  author!: string;
  albumTitle!: string;
  albumPublicationYear!: number;
  genre!: string;
  position?: number;
}
